examples/planning/train.ipynb

```diff
@@ -0,0 +1,332 @@
+   "metadata": {},
+   "source": [
+    "### Plot the train loss curve\n",
+    "We can plot the train loss against the iterations (batch-wise) to check if our model has converged."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.plot(np.arange(len(losses_train)), losses_train, label=\"train loss\")\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Store the model\n",
+    "\n",
+    "Let's store the model as a torchscript. This format allows us to re-load the model and weights without requiring the class definition later.\n",
+    "\n",
+    "**Take note of the path, you will use it later to evaluate your planning model!**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "to_save = torch.jit.script(model.cpu())\n",
+    "path_to_save = f\"{gettempdir()}/planning_model.pt\"\n",
+    "to_save.save(path_to_save)\n",
+    "print(f\"MODEL STORED at {path_to_save}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Congratulations in training your first ML policy for planning!\n",
+    "### What's Next\n",
+    "\n",
+    "Now that your model is trained and safely stored, you can evaluate how it performs in two very different situations using our dedicated notebooks:\n",
+    "\n",
+    "### [Open-loop evaluation](./open_loop_test.ipynb)\n",
+    "In this setting the model **is not controlling the AV**, and predictions are used to compute metrics only.\n",
+    "\n",
+    "### [Closed-loop evaluation](./closed_loop_test.ipynb)\n",
+    "In this setting the model **is in full control of the AV** future movements.\n",
+    "\n",
+    "## Pre-trained models\n",
+    "we provide a collection of pre-trained models for the planning task:\n",
+    "- [model](https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/models/planning_model_20201202003.pt) trained on the train.zarr for 15 epochs;\n",
+    "- [model](https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/models/planning_model_20201202002_early.pt) trained on the train.zarr for 2 epochs;\n",
+    "- [model](https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/models/planning_model_20201202002_nopt.pt) trained with perturbations disabled for 15 epochs;\n",
+    "- [model](https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/models/planning_model_20201202002_nopt_early.pt) trained with perturbations disabled for 2 epochs;\n",
+    "\n",
+    "We include two partially trained models to emphasise the important role of perturbations during training, especially during the first stage of training.\n",
+    "\n",
+    "To use the model just download the corresponding `.pt` file and use the local path in the evaluation notebooks"
```

Comment:
- . in end.
- What is the local path?
- I'd remove the before train.zarr.
- How about for models trained w/o perturbations also mention they were trained on train.zarr?